softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ProfileEntity.java

```diff
@@ -0,0 +1,108 @@
+package com.softgroup.common.dao.api.entities;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * Created by polev on 10.04.2017.
+ */
+@Entity
+@Table(name = "profiles")
+public class ProfileEntity extends BaseEntity implements Serializable {
+
+    @Column(name = "phone_number")
+    private String phoneNumber;
+
+    @Column(name = "create_date_time")
+    private Long createDateTime;
+
+    @Column(name = "update_date_time")
+    private Long updateDateTime;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "status")
+    private String status; //enum for online and offline
```

Comment:
@MariiaPolova this is not enum for online and offline )) This is for user status. Something like "I'm feeling fine!"